Sources/TuistGenerator/Generator/WorkspaceGenerator.swift

```diff
@@ -45,6 +45,8 @@ final class WorkspaceGenerator: WorkspaceGenerating {
     private let workspaceStructureGenerator: WorkspaceStructureGenerating
     private let cocoapodsInteractor: CocoaPodsInteracting
 
+    internal var fileHandler: FileHandling = FileHandler.shared
```

Comment:
We don't need to create an instance variable anymore because we can access `FileHandler.shared` anywhere from the body of the methods.

Reply:
Deleted 👍